xmodule/library_tools.py

```diff
@@ -27,43 +22,34 @@ def normalize_key_for_search(library_key):
 
 class LibraryToolsService:
     """
-    Service that allows LibraryContentBlock to interact with libraries in the
-    modulestore.
+    Service for LibraryContentBlock.
+
+    Allows to interact with libraries in the modulestore and blockstore.
+
+    Should only be used in the CMS.
     """
     def __init__(self, modulestore, user_id):
         self.store = modulestore
         self.user_id = user_id
 
-    def _get_library(self, library_key):
+    def get_latest_library_version(self, lib_key) -> str | None:
```

Comment:
get_library_version is now get_latest_library_version. Same functionality, just a more specific name.